SmartDeviceLink/SDLDisplayCapabilities.h

```diff
@@ -25,7 +25,14 @@ NS_ASSUME_NONNULL_BEGIN
  *
  * Required
  */
-@property (strong, nonatomic) SDLDisplayType displayType;
+@property (strong, nonatomic) SDLDisplayType displayType __deprecated_msg("See `displayName` instead");
+
+/**
+ The name of the connected display
+
+ Optional
+ */
+@property (strong, nonatomic) NSString *displayName;
```

Comment:
Shouldn't `displayName` be `nullable` since it is optional?

Reply:
Why yes it should